rpgJavaInterpreter-core/src/test/resources/overlay/MUTE12_02.rpgle

```diff
@@ -277,13 +277,15 @@
      C                   EVAL      DS0010=*HIVAL
     MU* VAL1(DS0010) VAL2(-9999999999,99999) COMP(EQ)
      C                   EVAL      DS0010=*LOVAL
-    MU* VAL1(DS0011) VAL2(9999) COMP(EQ)
+      * TODO incorrect, DS0011 is 2 bytes
+    MU* VAL1(DS0011) VAL2(99) COMP(EQ)
      C                   EVAL      DS0011=*HIVAL
-    MU* VAL1(DS0011) VAL2(-9999) COMP(EQ)
+    MU* VAL1(DS0011) VAL2(-99) COMP(EQ)
      C                   EVAL      DS0011=*LOVAL
-    MU* VAL1(DS0012) VAL2(999999999) COMP(EQ)
+      * TODO incorrect, DS0012 is 2 bytes
+    MU* VAL1(DS0012) VAL2(99) COMP(EQ)
```

Comment:
Is this comment still relevant?

Reply:
I would keep the comment as we changed the original test.